src/items/rules/skills.json

```diff
@@ -498,7 +498,7 @@
       "src": null,
       "system": {},
       "text": {
-        "content": "<p><strong>Source:</strong> CRB pg. 140</p><p>You are able to change your appearance to blend in and deceive others, whether to infiltrate</p><h2>Change Appearance</h2><p>You can use Disguise to change your appearance with 1d3 × 10 minutes of work with a disguise kit, by casting a spell such as disguise self, or by using a technological device such as a @UUID[Compendium.sfrpg.equipment.Item.BqC7LI0RlKsBmgAH]{holoskin}. The GM rolls the Disguise check in secret, so you’re not sure how good your disguise is. This check is opposed by the Perception check (see page 144) of anyone who might realize that you are not who you appear to be. If you are not drawing attention to yourself, other creatures do not usually get to attempt a Perception check to pierce your disguise. If creatures are being particularly alert for suspicious activity (such as security personnel on a starship or space station), it’s assumed that such observers are taking 10 on their Perception checks.</p><p>The effectiveness of your disguise depends on how much you’re changing your appearance. Without the aid of transformative magic or technology that allows you to do otherwise, you can use Disguise only to appear as a creature that is your size or one size category larger or smaller than you. This does not change your actual size or reach. Disguises are general—you cannot disguise yourself as a specific person. Certain magic spells, such as disguise self, grant you a +10 bonus to Disguise checks.</p><p>The DCs for Disguise checks are adjusted by the type of disguise as well as other circumstances determined by the GM. These modifiers are cumulative; use all that apply.</p><table><tbody><tr><th>DISGUISE</th><th>DC MODIFIER</th></tr><tr><td>Minor details altered only</td><td>–5</td></tr><tr><td>Major feature altered</td><td>+2 to +5</td></tr><tr><td>Disguised as a different race of the same creature type</td><td>+2 to +8</td></tr><tr><td>Disguised as a different creature type</td><td>+10</td></tr><tr><td>Disguised as a different size category</td><td>+10</td></tr></tbody></table>",
```

Comment:
This should include inline check buttons for the Disguise check and perception check.
Format: `@Check[type:disguise]`

Reply:
Did an exhaustive review of the Skills journal for links, inline checks and rolls